tests/guarded_transmute_bool_pedantic/mod.rs

```diff
@@ -22,21 +22,9 @@ fn just_enough() {
 #[test]
 fn invalid_bytes() {
     assert_eq!(guarded_transmute_bool_pedantic([0x00, 0x01, 0x02].as_ref()),
-               Err(Error {
-                   required: 1,
-                   actual: 3,
-                   reason: ErrorReason::InvalidValue,
-               }));
+               Err(Error::InvalidValue));
```

Comment:
I'm liking this syntax, much terser and doesn't spew meaningless data everywhere 👍

Reply:
:ok_hand: